### DISCUSSION TITLE
Global InputNumber parameter

### DISCUSSION THREAD

**Discussion by mtsh1000** [primefaces/community#165] [PrimeReact] [answered]
Hi, is it possible to set useGrouping: false for all INputNumber components globally?

**Comment by melloware** [comment]
Just wrap the component into your own component with that value defaulted.

For example here is where I wrap the Dialog component in my own component but set some defaults.

```ts
import { LazyLog } from "@melloware/react-logviewer";
import { Dialog, DialogProps } from "primereact/dialog";
import { ObjectUtils } from "primereact/utils";
import React, { useState } from "react";

const JobLogViewerDialog = (props: DialogProps) => {

	// local state
	const [log, setLog] = useState<string | undefined>(undefined);

	return (
		<Dialog maximizable maximized style={{ width: "100vw", height: "75vh" }} {...props}>
			<LazyLog text={log} selectableLines caseInsensitive enableSearch enableSearchNavigation enableHotKeys extraLines={1} />
		</Dialog>
	);
};

export default React.memo(JobLogViewerDialog);
```

so then I can use it anywhere in my code like...

```xml
 <JobLogViewerDialog header={header} visible={visibleLogViewer} onHide={() => setVisibleLogViewer(false)} />
```

**Reply by mtsh1000** [reply]
Thanks for the answer, that approach will work but I was wondering maybe PrimeReactProvider with config passthrough options could work?

**Reply by melloware** [reply]
Hmm have you tried it?

**Reply by mtsh1000** [reply]
Yes, I tried :
```js
{
  pt:{
    inputnumber:{
    useGrouping: false
    }
}
}
```

but it didn't work

**Reply by melloware** [reply]
I am not positive but I think passthrough only passes through DOM elements not to the core component itself?

**Reply by melloware** [reply]
Maybe request it as an enhancement ticket on GitHub?

**Reply by mtsh1000** [reply]
Ok, I will, thanks.

**Reply by melloware** [reply, marked as answer]
Reported here and marking as answer: https://github.com/primefaces/primereact/issues/5023
